python/extensions.bzl

```diff
@@ -32,10 +33,30 @@ def _python_impl(module_ctx):
                 register_coverage_tool = attr.configure_coverage_tool,
                 ignore_root_user_error = attr.ignore_root_user_error,
             )
+        for attr in mod.tags.host_python_interpreter:
+            _host_hub(name = "host_python_interpreter", interpreters = attr.interpreters, repo = attr.repo)
```

Comment:
I think we need to add a prefix to this name after all to support multiple Python versions. Today, with workspace, you can do e.g.

```
load("@python38//:defs.bzl", py38_interpreter = "interpreter")
load("@python310/:defs.bzl", py310_interpreter = "interpreter")
pip_parse(..., interpreter = py38_interpreter)
pip_parse(..., interpreter = py310_interpreter)
```

To get the equivalent, we'll need separately addressable repos for each python version.

This might also be a point in favor of moving it all into the toolchain() tag, because that call has the name attribute with the prefix that is traditionally used.

Reply:
I am setting the name.  This gives us a unique name for the interpreter.